static/css/marble.css

```diff
@@ -1151,6 +1168,10 @@ h6{
         margin-bottom: 30px;
     }
 
+    .header-card-title-margin{
```

Comment:
This is identical in all widths, move this out of the media widths.

Reply:
Moved this out of media queries.